components/molecule/photoUploader/src/InitialState.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Button from '@s-ui/react-atom-button'
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
+
+import {BASE_CLASS_NAME} from './config'
+const ALTERNATIVE_ACTION_TEXT = '- o -'
+
+const InitialState = ({buttonText, icon, text}) => {
+  return (
+    <div className={`${BASE_CLASS_NAME}-initialState`}>
+      <div className={`${BASE_CLASS_NAME}-iconInitialState`}>
+        <AtomIcon size={ATOM_ICON_SIZES.extraLarge}>{icon}</AtomIcon>
+      </div>
+      <div className={`${BASE_CLASS_NAME}-textState`}>
+        <span>{text}</span>
+        <span>{ALTERNATIVE_ACTION_TEXT}</span>
+      </div>
+      <div className={`${BASE_CLASS_NAME}-buttonState`}>
+        <Button size="small">{buttonText}</Button>
```

Comment:
we should use `color="primary"` as a prop in order to use the non deprecated buttons

Reply:
Fixed! Thanks a lot!